cli/internal/helm/charts/cilium/README.md

```diff
@@ -37,7 +37,7 @@ $ helm install cilium cilium/cilium --namespace=kube-system
 ```
 
 After Cilium is installed, you can explore the features that Cilium has to
-offer from the [Getting Started Guides page](https://docs.cilium.io/en/stable/gettingstarted/).
+offer from the [Getting Started Guides page](https://docs.cilium.io/en/latest/gettingstarted/).
```

Comment:
This link in the upstream cilium is broken, maybe we should just add it to the ignore list?

Reply:
Already done in .github/workflows/check-links.yml:29